docs: add install instructions

apkrelling opened the pull request.

apkrelling commented:
@callumrollo let me know if this looks OK. You might want to check the local pip install instructions - I'm not very familiar with that. Thanks!

callumrollo commented on README.md:
````diff
@@ -1,12 +1,68 @@
+conda env create --file environment.yml # create the environment 
+conda activate erddaplogs # activate the environment
+```
+or Mamba:  
````
I don't think the mamba instructions are necessary, those using mamba will know to substitute it for conda

callumrollo reviewed (changes requested):
Very thorough! I've added a couple of suggestions

callumrollo commented on README.md:
````diff
@@ -1,12 +1,68 @@
+# or as a developer, if you plan on contributing to the project:
+pip install -r requirements-dev.txt # install the dependencies
+```
+
````
should be a `pip install .` here at the end i think

callumrollo commented on README.md:
````diff
@@ -1,12 +1,68 @@
+* #### From pypi, using pip
+
+```sh
+pip install erddaplogs
````
move this one up to the top, simplest and cleanest install method

apkrelling commented:
Incorporated the suggestions:
- removed instructions for mamba
- moved install from pypi to the top
- changed from `pip install -r requirements-dev.txt` to `pip install .`
